health/QUICKSTART.md

```diff
@@ -87,7 +87,10 @@ lookup: average -1m percentage of used
 
 Let's look into each of the lines to see how they create a working health entity.
 
--   `alarm`: The name for your new entity. The name can be anything, but the only symbols allowed are `.` and `_`.
+-   `alarm`: The name for your new entity. The name needs to follow the requirements:
+     - Any alphabet letter or number.
+     - Symbols `.` and `_`.
+     - Cannot be `dimension name`, `chart name`, `family name` or `chart variable names`. 
```

Comment:
```suggestion
     - Cannot be `chart name`, `dimension name`, `family name`, or `chart variable names`. 
```

Reply:
I made all changes with the last commit, thank you!